jena-arq/src/main/java/org/apache/jena/riot/system/RiotLib.java

```diff
@@ -85,6 +85,16 @@ public static String blankNodeToIriString(Node node) {
         throw new RiotException("Not a blank node or URI");
     }
 
+    /**
+     * Convert a ARQ-encoded blank node URI nod to a blank node, otehrwise return the argument node unchanged.
```

Comment:
s/a ARQ/an ARQ if we say it like ark/arc? I actually say it that way but no idea what's the correct way.

And I think the `nod` can be deleted?

And s/otehrwise/otherwise

Reply:
Done